.github/actions/refresh-migration-database/action.yml

```
@@ -0,0 +1,43 @@
+name: Refresh migration DB
+description: Backup production DB and restore to migration DB
+
+inputs:
+  environment:
+    description: The name of the environment
+    required: true
+  azure-credentials:
+    description: Azure credentials
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - name: Install kubectl
+      uses: azure/setup-kubectl@v3
+
+    - name: Install konduit
+      shell: bash
+      run: make install-konduit
+
+    - name: Set AKS credentials (production)
+      shell: bash
+      run: az aks get-credentials -g s189p01-tsc-pd-rg -n s189p01-tsc-production-aks
+
+    - name: Backup production DB
+      shell: bash
+      run: |
+        bin/konduit.sh npq-registration-${{ inputs.environment }}-web -- pg_dump -E utf8 --compress=1 --clean --if-exists --no-privileges --no-owner --verbose -f backup-${{ inputs.environment }}.sql.gz
+
+    - name: Restore to migration DB
+      shell: bash
+      run: bin/konduit.sh -d s189p01-cpdnpq-mg-pg -k s189p01-cpdnpq-pd-app-kv -i backup-${{ inputs.environment }}.sql.gz -c -t 7200 npq-registration-${{ inputs.environment }}-web -- psql
```

```suggestion
      run: bin/konduit.sh -d s189p01-cpdnpq-mg-pg -k s189p01-cpdnpq-mg-app-kv -i backup-${{ inputs.environment }}.sql.gz -c -t 7200 npq-registration-${{ inputs.environment }}-web -- psql
```

Should this be migration keyvault?

---

Ah yeah that would make more sense! I'll move the secret into that keyvault